<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_map</class>
 <widget class="QWidget" name="form_map">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Global Water Maper (Map Visualize)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QTabWidget::tab:disabled { width: 0; height: 0; margin: 0; padding: 0; border: none; }

QPushButton{
 background-color: rgba(116, 121, 115, 95);

	color: rgb(255, 255, 255);
font: 11pt &quot;Arial&quot;;
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}

QPushButton:pressed {
background-color: rgba(116, 121, 115, 235);
color:rgba(255, 248, 230,255);
}

QPushButton:hover {
background-color: rgba(238, 234, 209, 125);
font: 12pt &quot;Arial&quot;;
}
QLabel{
background-color: rgba(211, 236, 231, 85);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QTextEdit{
background-color: rgba(241, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QDateEdit{
background-color: rgba(251, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QTableView{
background-color: rgba(108, 98, 88, 100);
color: rgb(255, 255, 255);
border-radius:10px;
}
QWebView{
background-color: rgba(241, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:10px;
}
QLineEdit{
background-color: rgba(241, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}
QComboBox{
background-color: rgba(251, 236, 231, 105);
color: rgb(255, 255, 255);
border-radius:5px;
border:0.5px solid rgba(211, 236, 231, 150);
}

QGroupBox{

border-width: 1px;
border-style: solid;
border-color: grey;
background-color: rgba(58, 54, 51, 70);
font: 11pt &quot;Arial&quot;;

color: rgb(255, 255, 255);
}
QTabWidget QWidget{
font: 10pt &quot;Bitstream Vera Sans&quot;;
color: rgb(255, 255, 255);
}

QTabBar::tab:hover {
border-top:1px solid rgb(111, 111, 111);
border-bottom:5px solid rgb(111, 111, 111);
font: 14pt &quot;Bitstream Vera Sans&quot;;
}
QTabBar::tab {
border-top:2px solid rgba(255, 248, 230,190);
font: 12pt &quot;Bitstream Vera Sans&quot;;
height: 25px; width: 150px;
}
QTabBar::tab:selected {
  border-top:4px solid rgba(166, 166, 166,90);
}


QTabWidget::pane
{
border-width: 1px;
border-style: solid;
border-color: grey;
background-color: rgba(58, 54, 51, 70);
}</string>
  </property>
  <widget class="QWebView" name="web_View">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>961</width>
     <height>481</height>
    </rect>
   </property>
   <property name="url">
    <url>
     <string>about:blank</string>
    </url>
   </property>
  </widget>
  <widget class="QTableView" name="tv_map">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>671</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>TABLE Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbx_table_name">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>user_input</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbx_schema">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>water_map</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>SCHEMA Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_goback">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>71</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Submited Data:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Abyssinica SIL</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string>Search:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tbx_search_csv">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Abyssinica SIL&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>web_View</zorder>
  <zorder>btn_load</zorder>
  <zorder>label_9</zorder>
  <zorder>tbx_table_name</zorder>
  <zorder>tbx_schema</zorder>
  <zorder>label_10</zorder>
  <zorder>btn_goback</zorder>
  <zorder>label_11</zorder>
  <zorder>label_14</zorder>
  <zorder>tbx_search_csv</zorder>
  <zorder>tv_map</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
